akka-bbb-apps/src/main/scala/org/bigbluebutton/core/apps/BreakoutModel.scala

```diff
@@ -18,8 +18,9 @@ object BreakoutModel {
       captureSlides: Boolean,
       captureNotesFilename: String,
       captureSlidesFilename: String,
+      includeMods: Boolean,

```

Comment:
I suggest to rename it to some clearer name! `sendInviteToModerators` would be fine.